app/services/reports/children_served_report_service.rb

```diff
@@ -39,12 +42,35 @@ def average_children_monthly
 
     # @return [Float]
     def per_child_monthly
+      total_distributions = organization.distributions.for_year(year).count
+      total_kits = organization.kits.count
+
+      total_avg = if total_distributions.zero? && total_kits.zero?
+        0.0
+      else
+        (distribution_average * total_distributions + kit_average * total_kits) / (total_distributions + total_kits)
+      end
+
+      total_avg.nan? ? 0.0 : total_avg
+    end
+
+    private
+
+    def distribution_average
```

Comment:
Let's make it clear that this distribution average  and the one for kits  are for disposables.    This is partly me knowing that we've got another change coming down the line to do something very similar for period products.